### ISSUE TITLE
R50 Zero-Shot on CC3M and CC12M

### ISSUE THREAD

**Issue by Hhhhhhao** [mlfoundations/open_clip#448]
Hi Team,

Thanks for this great work. I'm trying to replicate the reported performance of R50 on CC3M and CC12M, but I can only get ~18 and ~30 respectively, which has a large gap from the reported accuracy.
<img width="594" alt="image" src="https://user-images.githubusercontent.com/39571115/221219727-5c5e56bb-6832-42c0-b14a-db367c9a7c66.png">

I'm using the command in README and tried to adjust the learning rate and batch size, but I didn't work. 
I wonder if this is a data issue. I have 2806772 samples in CC3M and  10062127 in CC12M.
I also want to ask is it necessary to scale the learning rate according to batch size? The paper used 5e-4 and 32768 as batch size, if I use smaller batch size, is it expected to scale the learning lower too?

Thanks in advance.

**Comment by mitchellnw** [comment]
Hello,

These are pretty close to the reported numbers (~1%), so my guess is that it's just a minor hparams difference or that you have less data as more links have expired.

For CC3M we get closer to 19 with the hparams:
    --batch-size=128 
    --lr=1e-3 
    --wd=0.1 
    --epochs=30 
    --model RN50
 on 8 GPUs, so global batch 1024.
 
 I don't think we've tried LR below 5e-4, but I'd guess that's too low.
 
 Hope that helps!

**Comment by Hhhhhhao** [comment]
Thanks, 

I can got 18.53% with this parameter setting, trained on CC3M. I think this result is fine considering the expired links. 
I will try CC12M in the next.

**Comment by mitchellnw** [comment]
great, thanks, feel free to re-open issue if anything seems wrong
